e2e/js_run_devserver/test.sh

```diff
@@ -1,18 +1,26 @@
 #!/usr/bin/env bash
 set -o errexit -o nounset -o pipefail
 
-BZLMOD_FLAG="${BZLMOD_FLAG:-}"
-
 # Integration test for js_run_devserver run with ibazel
 
-# shellcheck disable=SC2086
-bazel run $BZLMOD_FLAG -- @pnpm//:pnpm --dir "$PWD" install
+BZLMOD_FLAG="${BZLMOD_FLAG:---enable_bzlmod=1}"
+
+bazel run "$BZLMOD_FLAG" -- @pnpm//:pnpm --dir "$PWD" install
 
 ./serve_test.sh //src:serve
 ./serve_test.sh //src:serve_alt
 ./serve_test.sh //src:serve_simple
 ./serve_test.sh //src:serve_simple_bin
 ./serve_test.sh //src:serve_naughty
 ./serve_test.sh //src:serve_naughty_bin
-./multirun_test.sh //src:serve_multi
+
+bazel_version=$(head -n 1 .bazelversion)
+bazel_major=${bazel_version::1}
+if [[ "$bazel_major" -eq 7 ]]; then
```

Comment:
nit: we'll have to update this line again for Bazel 8

Reply:
updated to `-ge`